testing/src/main/scala/org/http4s/testing/ArbitraryInstances.scala

```diff
@@ -861,6 +861,9 @@ trait ArbitraryInstances {
         body <- http4sTestingGenForPureByteStream
       } yield Response(status, httpVersion, headers, body)
     }
+
+  implicit val arbUUID: Arbitrary[UUID] =
```

Comment:
I would rather put this in another module of arbitraries, like we did for fs2.  I don't like providing instances of types we don't own for typeclasses we don't own into public API.

I'm pretty surprised this isn't core scalacheck functionality.

Reply:
> I would rather put this in another module of arbitraries, like we did for fs2.

Can you please say more? I don't follow on how to implement that. My guess was just adding it to `Http4sSpec` like https://github.com/http4s/http4s/blob/series/0.20/testing/src/test/scala/org/http4s/Http4sSpec.scala#L62, but I'm not sure.

I pushed up https://github.com/http4s/http4s/pull/3167/commits/4329e58da7e6cda204f068cd9d78b0d5c0d3079b, but please let me know if it does not address your comment, Ross.

Reply:
That's fine.  We added [fs2Arbitraries](https://github.com/http4s/http4s/blob/master/testing/src/test/scala/org/http4s/testing/fs2Arbitraries.scala) in 0.21, which I think is a good pattern for these.  But the important thing here is that they're not in `src/main`.  We don't want other people depending on ours and someone else's.  If we keep it private, and Scalacheck adds one later, we can delete ours without affecting anyone else.

Reply:
Roger that, Ross. I wasn't aware that `Arbitraries`, where I originally added the `Arbitrary[UUID]` lives at `src/main`. I wrongly assumed it was `src/test`.